tests/end_to_end/enums/irix-o2-out.c

```diff
@@ -0,0 +1,14 @@
+enum my_enum array[4] = { THREE, TWO_TOO, TWO, ZERO }; /* const */
+
+enum my_enum test(enum my_enum x) {
+    switch (x) {                                    /* irregular */
+    case ZERO:
+        return *array;
+    case TWO:
+        return *array;
+    case THREE:
+        return *array;
+    default:
+        return ZERO;
+    }
+}
```

Comment:
This test shows an outstanding issue with `%hi`/%lo`, where the `+4` part of `%lo(array+4)` is discarded, so the output here is incorrect. 
(The asm was hand-crafted, but the original asm had `0xE4` instead of `%lo(array+4)`; afaict my edits aren't the cause of this issue)


This test does showcase all 3 places enum names can be used:
- As part of a data declaration (in `array[4] = ...` on line 1)
- As part of a switch label
- As a `Literal` (in `return ZERO;` on line 12)